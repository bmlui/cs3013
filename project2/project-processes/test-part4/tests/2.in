sleep 30 &
sleep 20 &
sleep 10 &
jobs
exit

