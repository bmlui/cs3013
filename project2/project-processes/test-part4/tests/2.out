test-part4$ sleep 30 &
[1]
test-part4$ sleep 20 &
[2]
test-part4$ sleep 10 &
[3]
test-part4$ jobs
1: sleep 30 
2: sleep 20 
3: sleep 10 
test-part4$ exit
